openedx/core/lib/blockstore_api/methods.py

```diff
@@ -44,7 +47,11 @@ def toggle_blockstore_api(func):
     def wrapper(*args, **kwargs):
         if use_blockstore_app():
             return getattr(blockstore_api_methods, func.__name__)(*args, **kwargs)
-        return func(*args, **kwargs)
+        joined_args = " "
+        log.Info('blockstore ' + func.__name__ + ' API call called with ' + joined_args.join(args) + ' arguments')
+        ret_object = func(*args, **kwargs)
+        log.Info('blockstore ' + func.__name__ + ' API call is done')
+        return ret_object
```

Comment:
@bszabo This logging will only happen for wrapped calls to the blockstore _service_, but what we're testing by flipping that waffle switch is the blockstore _app_. If you want to log the calls to the app, you'll need to move these log messages to the line above:

```
         if use_blockstore_app():
            return getattr(blockstore_api_methods, func.__name__)(*args, **kwargs)
```